modmail/bot.py

```diff
@@ -35,11 +36,14 @@ class ModmailBot(commands.Bot):
     """
 
     logger: ModmailLogger = logging.getLogger(__name__)
+    tickets: t.Dict[int, Ticket] = dict()
 
     def __init__(self, **kwargs):
         self.config = CONFIG
         self.start_time: t.Optional[arrow.Arrow] = None  # arrow.utcnow()
         self.http_session: t.Optional[ClientSession] = None
+        self._guild_available = asyncio.Event()
```

Comment:
Why _?

Reply:
Because the `wait_until_guild_available()` coroutine should be used instead. This is a private property.